dotnet/README.md

```diff
@@ -0,0 +1,76 @@
+# .NET CSFLE Example
+
+## Windows Only
+
+Currently, remote KMS support is Windows only.
+
+## Steps
+
+1. Clone this repository:
+
+   ```
+     git clone https://github.com/mongodb-university/csfle-guides.git
+   ```
+
+   Navigate to the CSFLE directory. Work from this directory for the remainder of these
+   instructions.
+
+2. Start a `mongod` instance (Enterprise version >= 4.2) running on port 27017
+
+3. Make sure you have the `master-key.txt` file in the root of your
+   execution environment if you are using a local KMS. This is a 96-byte cryptographically-secure generated
+   master encryption key required to run this example project. To generate your
+   own master key or use a KMS, refer to the [CSFLE Use Case
+   Guide](https://docs.mongodb.com/ecosystem/use-cases/client-side-field-level-encryption-guide/).
+
+4. Configure variables for your preferred remote KMS, if
+   necessary. Out of the box, this application looks for KMS credentials and data key options
+   via the following environmental variables:
+
+   - AWS
+
+     - KMS Credentials
+
+       - FLE_AWS_ACCESS_KEY
+       - FLE_AWS_SECRET_ACCESS_KEY
+
```

Comment:
In the doc https://docs.google.com/document/d/18rO1QYORNvKs8rYWtXKdrT-2YaORMJ-r8xcM8g1SJ8Q/edit# under "New Key Encryption Key (masterKey) options", there's an optional endpoint specification. Is this available in the C# version?

Reply:
It is included in the Master Key Options portion